pages/add/form-fields/detail-info-fields.js

```diff
@@ -0,0 +1,108 @@
+module.exports = {
+  creators: {
+    type: `text`,
+    label: `Who are the creators? This could be staff, contributors, partners…`,
+    placeholder: `Name`,
+    fieldClassname: `form-control`,
+    multiplicity: 1,
+    addLabel: `+ Add another`,
+    // optional: true
+  },
+  interest: {
+    type: `text`,
+    label: `Why might this be interesting to other people in our network?`,
+    placeholder: ``,
+    fieldClassname: `form-control`,
+    validator: {
+      error: `Maximum 300 characters.`,
+      validate: function(value) {
+        if (!value) {
+          // this is an optional field, it can be left blank
+          return;
+        }
+
+        return value && value.length > 300; // return true if value fails validation
+      }
+    },
+  },
+  tags: {
+    type: `text`,
+    label: `Keywords to help with search by program, event, campaign, subject …`,
+    placeholder: `#mozfest  #code  #tool`,
+    fieldClassname: `form-control`
+  },
+  issues: {
+    type: `checkboxGroup`,
+    label: `Check any Key Internet Issues that relate to your project.`,
+    options: [ `Online Privacy & Security`, `Open Innovation`, `Decentralization`, `Web Literacy`, `Digital Inclusion` ],
```

Comment:
I'd like to tie this to the api server's `/issues?format=json` but definitely not now